index.html

```diff
@@ -17,34 +17,34 @@ <h2>Mark</h2>
     </header> 
 
     <section class="landing-page all-body" id="body-cover">
-               
-                <h1 class="title">I'm Mark <br>Glad to see you!</h1>
-                <p class="intro">
-                    I'm a software developer! I can help you build a product , 
-                    feature or website Look through some of my work and experience! 
-                    If you like what you see and have a project you need coded, don't 
-                    hestiate to contact me.
-                </p>
-            
-                <p class="lets-connect">LET'S CONNECT</p>
-                    
-                <ul class="social-buttons">
-                    <li><a href="#"><img src="images/Vector.svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Group.svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Vector (3).svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Vector (4).svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Vector (5).svg" class="icons"></a></li>
-                </ul>
+               <div class="ctn">
+                    <h1 class="title">I'm Mark <br>Glad to see you!</h1>
+                    <p class="intro">
+                        I'm a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience! 
+                        If you like what you see and have a project you need coded, don't 
+                        hestiate to contact me.
+                    </p>
                 
+                    <p class="lets-connect">LET'S CONNECT</p>
+                        
+                    <ul class="social-buttons">
+                        <li><a href="#"><img src="images/Vector.svg" class="icons"></a></li>
+                        <li><a href="#"><img src="images/Group.svg" class="icons"></a></li>
+                        <li><a href="#"><img src="images/Vector (3).svg" class="icons"></a></li>
+                        <li><a href="#"><img src="images/Vector (4).svg" class="icons"></a></li>
+                        <li><a href="#"><img src="images/Vector (5).svg" class="icons"></a></li>
+                    </ul>
+               </div>
     </section>
 
 
     <section class="works-container" id="works-container">
         <div class="works">
-            <img src="images/Snapshoot Portfolio.svg" alt="" class="tonic">
-            <img src="images/Snapshoot Portfolio1.svg" alt="" class="tonic-1">
-            <div class="div1">
-                <h2 class="header-tonic">Tonic</h2>
+            <section class="illustration">
+            </section>
+            <div class="div1 w_ctn">
+                <h2 class="header">Tonic</h2>
```

Comment:
- [ ] Kindly style the card well by adding some more padding to the `h2`
- Your Design
<img width="1440" alt="Screenshot 2023-05-12 at 11 25 58" src="https://github.com/MarkYegon7/Portfolio-Setup/assets/81444338/179530e9-d707-4ce9-a7e7-8483bb59b34c">

- Figma Design
<img width="1348" alt="Screenshot 2023-05-12 at 11 29 24" src="https://github.com/MarkYegon7/Portfolio-Setup/assets/81444338/de3f39a5-e6e6-4f5c-a913-54dc2d93e7be">

Reply:
![Screenshot from 2023-05-13 00-12-38](https://github.com/MarkYegon7/Portfolio-Setup/assets/85193476/663a74d1-3b7c-42c1-bc2e-6e02538ca992)

Padding is correctly placed